app/logic/events.py

```diff
@@ -121,7 +120,7 @@ def getUpcomingEventsForUser(user,asOf=datetime.now()):
                             .where(Event.startDate >= asOf)
                             .where(Event.timeStart > asOf.time())
                             .distinct() # necessary because of multiple programs
-                            .order_by(Event.startDate)
+                            .order_by(Event.eventName) # keeps the order of events the same when the dates are the same
```

Comment:
We still need them ordered by date, just ordered by event name after date. I would think.